configuration/scripts/cice.batch.csh

```diff
@@ -91,7 +91,20 @@ cat >> ${jobfile} << EOFB
 #PBS -l walltime=${batchtime}
 EOFB
 
-else if (${ICE_MACHINE} =~ thunder* || ${ICE_MACHINE} =~ gordon* || ${ICE_MACHINE} =~ conrad*  || ${ICE_MACHINE} =~ gaffney* || ${ICE_MACHINE} =~ koehr*) then
+else if (${ICE_MACHINE} =~ gordon*) then
+if (${runlength} > 0) set queue = "standard"
```

Comment:
This is not a good idea.  It will always overwrite any queue set by the user.  We should either change the default queue on gordon (and conrad) to standard or the user can set --queue standard on the cice.setup command line.  I always do that when I'm running a test suite.

Reply:
I reverted to original cice.batch.csh configuration. I will use the --queue option for testing on gordon and conrad.